src/definitions/modules/tab.js

```diff
@@ -100,9 +100,9 @@ $.fn.tab = function(parameters) {
             initializedHistory = true;
           }
 
-          if(instance === undefined && module.determine.activeTab() == null) {
+          if(settings.autoTabActivation && instance === undefined && module.determine.activeTab() == null) {
             module.debug('No active tab detected, setting first tab active', module.get.initialPath());
-            module.changeTab(module.get.initialPath());
+            module.changeTab(settings.autoTabActivation === true ? module.get.initialPath() : settings.autoTabActivation);
```

Comment:
From the PR to docs,

> When a string is given and no active tab exists, the tab those named path matches the string will be activated 
I see the reason.

Reply:
The verbose === is needed to also check for type, so only a boolean `true` will do the same as before and will guess the tab. If it's a string otherwise (`if(settings.autoTabActivation)` will also be true then!) then it uses that string to activate a tab. That's why